akka-sample-persistence-dc-scala/README.md

```diff
@@ -0,0 +1,19 @@
+akka-sample-persistence-dc-scala
+================================
+
+This is an example project demonstrating [Akka Persistence Multi-DC](https://developer.lightbend.com/docs/akka-commercial-addons/current/persistence-dc/index.html)
+which is available as a commercial feature as part of the [Lightbend Subscription](https://www.lightbend.com/subscription).
+
+## How to run
+
+1. In terminal 1: `sbt "runMain sample.persistence.multidc.ThumbsUpApp cassandra"`
+
+1. In terminal 2: `sbt "runMain sample.persistence.multidc.ThumbsUpApp 2551 eu-west"`
+
+1. In terminal 3: `sbt "runMain sample.persistence.multidc.ThumbsUpApp 2552 eu-central"`
+
+1. In terminal 4:
+    * To add a thumbs-up for resource `akka` from user `u1` in DC `eu-west`: `curl -X POST http://0.0.0.0:22551/thumbs-up/akka/u1`
+    * To add a thumbs-up for resource `akka` from user `u2` in DC `eu-west`: `curl -X POST http://0.0.0.0:22552/thumbs-up/akka/u2`
+    * To get the users that gave thumbs-up for resource `akka`: `curl http://0.0.0.0:22552/thumbs-up/akka`
```

Comment:
curl to `0.0.0.0`, does that really work? Would `127.0.0.1` be safer?

Reply:
works for me, but  127.0.0.1 is better